Source/Private/ASRectMap.mm

```diff
@@ -62,7 +62,7 @@ - (id)copyWithZone:(NSZone *)zone
 
   // { ptr1->rect1 ptr2->rect2 ptr3->rect3 }
   NSMutableString *str = [NSMutableString string];
-  for (auto it = _map.begin(); it != _map.end(); it++) {
+  for (var it = _map.begin(); it != _map.end(); it++) {
```

Comment:
Should we use `for (let &e : _map) { e.first; e.second }`? It should provide exact same performance with clearer code.

Reply:
Yep agree - use ref and for each. Changed.